posts/2024-03-04_inside_the__phar.../inside_the__pharmaverse.qmd

```diff
@@ -0,0 +1,136 @@
+Council meeting minutes are published in the Pharmaverse GitHub [here](https://github.com/pharmaverse/pharmaverse/tree/main/static/council%20meeting%20minutes).
+
+## Package Inclusion
+
+Applications for a package to be included in the Pharmaverse may originate from anybody and anywhere.
+The Council (through its [Working Groups](https://pharmaverse.org/contribute/wg/)) reviews applications and either (i) approves the package into the Pharmaverse or (ii) rejects the package with a rationale.
+When a rejection has been issued, common reasons have been ‘not now’ with feedback to enhance testing, documentation, or other aspects that enrich the package.
+Working Groups may also steer the applicant to other packages with similar functionality.
+In this sense, respective package maintainers (product owners) are encouraged to work collaboratively to reduce duplicated features across packages and improve the overall experience of the user base.
+The Council and Working Groups are tasked with assessing this aspect of the Pharmaverse code base, but the decision on implementation ultimately resides with the individual package maintainers.
+
+Package reviews are conducted in the open:
+
+-   **Example 1:** {riskassessment} from R Validation Hub working group – result successful: <https://github.com/pharmaverse/pharmaverse/issues/195>
+
+-   **Example 2:** {rhino} from Appsilon – result successful: <https://github.com/pharmaverse/pharmaverse/issues/260>
+
+-   **Example 3:** {synthetic.cdisc.data} from Roche – result unsuccessful: <https://github.com/pharmaverse/pharmaverse/issues/235>
+
+At the time of publication, there were over 250 contributors to Pharmaverse packages, across a network of organizations much broader than just the 5 Council member organizations.
+Pharmaverse Council encourages diversity of individual contributors and the organizations they are affiliated with.
+The decision on product development team rosters is the sole discretion of the packages.
+Of course, Council member organizations are providing people to support the Pharmaverse ecosystem (e.g., developers and product owners).
+However, both strategic and operational decisions on the development and maintenance of individual packages reside outside the Pharmaverse Council.
+
+## User adoption
+
+Pharma companies are free to choose any selection of R packages for clinical reporting and only they can justify their choices.
+The [Pharmaverse website](https://pharmaverse.org/) states that “\[a\]nyone is free to choose any selection of Pharmaverse recommended software or those from any other source.” In addition, Pharmaverse does not seek to engage or get endorsement from any health authority.
+We are merely trying to provide a public service to help individuals and organizations involved in clinical reporting navigate a vast field of available R packages licensed as open source.
+
+## So, what really is Pharmaverse?
```

Comment:
What about the example site and showing how we take pharamverse and other 
 packages to build our common deliverables (ADaMs, TLFs. xpts, etc)

Reply:
I will add a simp;le sentence to connect the reader